### ISSUE TITLE
No ability to take instance out of "Read-only" mode and permissions not saved anywhere.

### ISSUE THREAD

**Issue by CapnRedB** [baine-reynolds/BBS_Read-Only#1]
If the user decides to undo this, there is no way to do so other than manually. And the permissions aren't saved so if they do it manually they have no way of tracking the old permissions.

**Comment by baine-reynolds** [comment]
Good point. When I get a chance i'll add a "--unlock" option to reverse this.  As for tracking, I'll still have to come up with something.
